test/solidity-ibc/ICS20TransferTest.t.sol

```diff
@@ -80,134 +71,54 @@ contract ICS20TransferTest is Test {
         assertEq(sequence, 42);
     }
 
-    function test_failure_sendTransfer() public {
-        IICS26RouterMsgs.Packet memory packet = _getTestPacket();
-
-        // just to make sure it doesn't accidentally revert on the router call
-        vm.mockCall(address(this), abi.encodeWithSelector(IICS26Router.sendPacket.selector), abi.encode(uint32(42)));
-
-        vm.startPrank(sender);
+    function test_newMsgSendPacketV2() public {
+        address senderAddress = makeAddr("my-sender");
+        string memory expectedSenderStr = Strings.toHexString(senderAddress);
+        IICS20TransferMsgs.ERC20Token[] memory erc20Tokens = new IICS20TransferMsgs.ERC20Token[](1);
+        erc20Tokens[0] = IICS20TransferMsgs.ERC20Token({ contractAddress: address(erc20), amount: defaultAmount });
 
         IICS20TransferMsgs.SendTransferMsg memory msgSendTransfer = IICS20TransferMsgs.SendTransferMsg({
-            denom: erc20AddressStr,
-            amount: defaultAmount,
-            receiver: receiverStr,
-            sourceClient: packet.sourceClient,
-            destPort: packet.payloads[0].sourcePort,
-            timeoutTimestamp: uint64(block.timestamp + 1000),
-            memo: "memo"
+            tokens: erc20Tokens,
+            receiver: "my-receiver",
+            sourceClient: "my-source-client",
+            destPort: "my-dest-port",
+            timeoutTimestamp: 123,
+            memo: "my-memo",
+            forwarding: IICS20TransferMsgs.Forwarding({ hops: new IICS20TransferMsgs.Hop[](0) })
         });
 
-        // just to prove that it works with the unaltered transfer message
```

Comment:
I deleted a bunch of tests that were already covered in the integration tests, to keep the maintenance burden down.